lib/queries/batchInsert.js

```diff
@@ -1,22 +1,26 @@
 import valueSubQuery from './valueSubQuery';
 import batchParameter from './batchParameter';
+import returningQuery from './returningQuery';
 
-export default ({ table, fields, returnFields = ['*'] }) => (entities) => {
-    if (!entities || entities.length === 0) {
-        return { sql: '' };
-    }
 
-    const parameters = batchParameter(fields)(entities);
-    const getValueSubQuery = valueSubQuery(fields);
+export default ({ table, fields, returnFields }) => {
+    const returning = returningQuery(returnFields);
 
-    const returning = returnFields.join(', ');
+    return (entities) => {
+        if (!entities || entities.length === 0) {
+            return { sql: '' };
```

Comment:
An additional check in the client might be nice to avoid running empty string queries ?